flit/wheel.py

```diff
@@ -35,6 +35,11 @@ def _write_wheel_file(f, *, supports_py2=False):
     f.write("Tag: py3-none-any\n")
 
 
+def _set_zinfo_mode(zinfo, mode):
+    # 0xFFFF = “regular file”
+    zinfo.external_attr = mode << 16
```

Comment:
i had to find this: https://stackoverflow.com/questions/434641 to understand why `<< 16` was here.   maybe a comment?

Reply:
It’s in the line above, but maybe a bit cryptic, I improved it.